lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -39,113 +39,226 @@
+            ${_("Starts - {start_date}").format(start_date=course.start_datetime_text("SHORT_DATE"))}
+          % endif
+          </span>
+        </div>
+        % if show_courseware_link:
+          <div class="wrapper-course-actions">
+            <div class="course-actions">
+              % if course.has_ended():
+               % if not is_course_blocked:
+                <a href="${course_target}" class="enter-course archived">${_('View Archived Course')}</a>
+              % else:
+                <a class="enter-course-blocked archived">${_('View Archived Course')}</a>
+              % endif
+              % else:
+              % if not is_course_blocked:
+                <a href="${course_target}" class="enter-course">${_('View Course')}</a>
+              % else:
+                <a class="enter-course-blocked">${_('View Course')}</a>
+              % endif
+              % endif
 
+              <!-- <a href="#" data-tooltip="${_('Share on Facebook')}" class="action action-facebook" aria-haspopup="true" aria-expanded="false">
+                <span class="sr">Facebook</span>
+                <i class="fa fa-facebook" aria-hidden="true"></i>
+              </a> -->
+              <!-- <a href="#" data-tooltip="${_('Share on Twitter')}" class="action action-twitter" aria-haspopup="true" aria-expanded="false">
+                <span class="sr">Twitter</span>
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+              </a> -->
+              <div class="wrapper-action-more">
+                <a href="#actions-dropdown-${dashboard_index}" class="action action-more" id="actions-dropdown-link-${dashboard_index}" aria-haspopup="true" aria-expanded="false" data-course-number="${course.number | h}" data-dashboard-index="${dashboard_index}">
+                  <span class="sr">Course options dropdown</span>
+                  <i class="fa fa-cog" aria-hidden="true"></i>
+                </a>
+                <div class="actions-dropdown" id="actions-dropdown-${dashboard_index}">
+                  <ul class="actions-dropdown-list" id="actions-dropdown-list-${dashboard_index}" aria-label="More Options" role="menu">
+                    <li class="actions-item" id="actions-item-unenroll-${dashboard_index}">
+                      % if is_paid_course and show_refund_option:
+                        ## Translators: The course name will be added to the end of this sentence.
+                        % if not is_course_blocked:
+                        <a href="#unenroll-modal" class="action action-unenroll" rel="leanModal" data-course-id="${course.id | h}" data-course-number="${course.number | h}" data-dashboard-index="${dashboard_index}" onclick="document.getElementById('track-info').innerHTML='${_("Are you sure you want to unenroll from the purchased course")}';
+                        document.getElementById('refund-info').innerHTML=gettext('You will be refunded the amount you paid.')">
```

Comment:
I've noted this comment and debt on a ticket here: https://openedx.atlassian.net/browse/TNL-1830 

thanks for flagging this!